### ISSUE TITLE
Priority of :> not handled correctly

### ISSUE THREAD

**Issue by oggy-** [will62794/tla-web#34]
Hi, first off thanks for the project, this is very nice! However, there seems to be a bug in the interpreter, in handling the priority of the `:>` operator. It gets confused by this spec:
```
---- MODULE Web ----
EXTENDS TLC

VARIABLES
    x,
    y

Init ==
    /\ x = 0 :> 0
    /\ y = 1 :> 1

Next == UNCHANGED <<x, y>>

====
```
But this works:
```
---- MODULE Web ----
EXTENDS TLC

VARIABLES
    x,
    y

Init ==
    /\ x = (0 :> 0)
    /\ y = (1 :> 1)

Next == UNCHANGED <<x, y>>

====
```

**Comment by will62794** [comment]
Thanks for reporting this, I will take a look. It seems this may be related to some odd behavior of the underlying parser, but not entirely sure yet. Posted initial question related [here](https://github.com/tlaplus-community/tree-sitter-tlaplus/issues/87).

**Comment by will62794** [comment]
@oggy Seems that the underlying issue had to do with a quirk of [the parser](https://github.com/tlaplus-community/tree-sitter-tlaplus). Was discussed and addressed [here](https://github.com/tlaplus-community/tree-sitter-tlaplus/issues/87), and the web interpreter is now updated to include this fix (e15e137). See working example of original spec snippet [here](https://will62794.github.io/tla-web/#!/home?specpath=./specs/with_state_graphs/simple_tlc_fn.tla).
